earth_enterprise/src/fusion/autoingest/sysman/DependentStateTree.cpp

```diff
@@ -152,7 +155,7 @@ void DependentStateTreeFactory::FillInVertex(
   }
   // If I'm in the dependency tree I need to add my dependents because they are
```

Comment:
Might want to update this comment, though I'm not sure quite how to word it.

Reply:
Updated. I'm open to suggestions on the wording.